crates/sui-node/src/lib.rs

```diff
@@ -732,19 +733,16 @@ impl SuiNode {
         );
 
         loop {
-            let committee = checkpoint_executor
-                .run_epoch(self.state.epoch_store().clone())
-                .await;
+            let cur_epoch_store = self.state.load_epoch_store_one_call_per_task();
```

Comment:
suggestion: Pass the initial epoch store in to monitor_reconfiguration. Then have reconfigure_state return the new one, and update it there.

Reply:
Done